content/docs/1.7.0/advanced-resources/os-distro-specific/container-optimized-os-support.md

```diff
@@ -0,0 +1,69 @@
+---
+title:  Container-Optimized OS Support
+weight: 5
+---
+
+## Requirements
+
+You must meet the following requirements before installing Longhorn on a Container-Optimized OS (COS) cluster.
```

Comment:
Where are the requirements? We cannot keep this sentence if the requirements are not mentioned at all.

Reply:
The requirements refer to the following section about the dependencies. Do you have any alternative suggestions for how we can convey this better?

Reply:
I've moved part of the dependency section to the requirement section. Let me know what you think.

Reply:
Got it. The changes that you made work for me.